src/data/ellpack_page.cu

```diff
@@ -302,6 +301,35 @@ EllpackPageImpl::EllpackPageImpl(AdapterT* adapter, float missing, bool is_dense
 ELLPACK_SPECIALIZATION(data::CudfAdapter)
 ELLPACK_SPECIALIZATION(data::CupyAdapter)
 
+
+template <typename AdapterBatch>
+EllpackPageImpl::EllpackPageImpl(AdapterBatch batch, float missing, int device,
+                                 bool is_dense, int nthread, bool is_row_major,
+                                 common::Span<size_t> row_counts_span,
+                                 size_t row_stride, size_t n_rows, common::HistogramCuts cuts) {
+  dh::safe_cuda(cudaSetDevice(device));
+
+  *this = EllpackPageImpl(device, cuts, is_dense, row_stride, n_rows);
+  if (is_row_major) {
+    CopyDataRowMajor(batch, this, device, missing);
+  } else {
+    // CopyDataColumnMajor(adapter, batch, this, missing);
+    LOG(FATAL) << "Not implemented";
```

Comment:
What makes the second `EllpackPageImpl` different from the first `EllpackPageImpl`, such that column major is not supported?